core/server/data/schema/schema.js

```diff
@@ -112,6 +112,16 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    oauth: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        provider: {type: 'string', maxlength: 191, nullable: false},
+        provider_id: {type: 'string', maxlength: 191, nullable: false},
```

Comment:
We'd need some references regarding this field because at the moment it's unclear if 191 characters would do a trick.

Reply:
@naz I've updated the issue with a justification for each length + rebased it, it should be ready to be merged, let me know!